### DISCUSSION TITLE
scanorama loading in R

### DISCUSSION THREAD

**Discussion by gmoore5** [brianhie/scanorama#89] [Q&A]
Hi im new to coding. 

When i load reticulate in R and try to import scanorama I get an:

"> scanorama <- import('scanorama')
Error in py_module_import(module, convert = convert) : 
  ModuleNotFoundError: No module named 'scanorama'"

I have the latest python, reticulate, R, and scanorama:

> py_config()
python:         /Users/haimmoore/Library/r-miniconda/envs/r-reticulate/bin/python
libpython:      /Users/haimmoore/Library/r-miniconda/envs/r-reticulate/lib/libpython3.6m.dylib
pythonhome:     /Users/haimmoore/Library/r-miniconda/envs/r-reticulate:/Users/haimmoore/Library/r-miniconda/envs/r-reticulate
version:        3.6.13 | packaged by conda-forge | (default, Feb 19 2021, 05:36:16)  [GCC Clang 11.0.1]
numpy:          /Users/haimmoore/Library/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/numpy
numpy_version:  1.19.5

**Comment by brianhie** [comment]
Can you check if scanorama is successfully installed in your current environemnt? You can type "python" in the command line to get a python shell. Then, just see if "import scanorama" succeeds without throwing an error.

**Comment by gmoore5** [comment]
I did not get any error when i tried that


The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
(base) Haims-MacBook-Pro:~ haimmoore$ python
Python 3.8.8 (default, Apr 13 2021, 12:59:45) 
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import scanorama
>>>

**Comment by brianhie** [comment]
Can you try explicitly pointing R to your particular Python instance? Get the path via the “which python” command, then set the path as described here: https://stackoverflow.com/questions/62501000/cannot-import-python-modules-on-r-studio
